lab0/INSTRUCTIONS.md

```diff
@@ -1,18 +1,20 @@
 Lab 0: Building your first Nerdpack
 ===================================
 
-The purpose of this lab is to provide hands on experience building on top of the New Relic One platform. Following the instructions in this lab, you will be up and running quickly. First, let's make sure you understand a few key concepts that you'll be working with.
+The purpose of this lab is to provide you a quick, hands on experience building on top of the New Relic One platform. First, let's make sure you understand a few key concepts that you'll be working with.
```

Comment:
A few comments here:

1. Instead of discussing _what this lab will do for you_, we should talk about _what the user will do and gain from it_.
2. Try to steer away from vague terms like "quick". It makes an assumption about how fast the user will gain something from this lab. If it's necessary, you can say something like, "In three steps" or "In five minutes" but "quick" is vague.
3. Our guides should be user focused. So they should use the second person persepective when talking about the user: "First, ~let's~ make sure you understand..."
4. "Understand" is a loaded term. A definition hardly yields a true understanding. So instead of "understand", we can just tell the user what we're giving them. Better would be: "here are some definitions"
5. We also don't have to mention "In this lab" if we can help it. The reader knows that we're talking about the lab here... because they're reading it :)

References:

- [You vs Let's/We](https://docs.newrelic.com/docs/style-guide/writing-guidelines/voice-strategies-docs-sound-new-relic/#clamshell-2)

Recommended change:

> New Relic is a programmable platform. This means you can write and deploy your own code to customize how you see your data, pull in data from third-party services, and create experiences that let you get more from New Relic than what's offered by default. Before you write any code, it's important to learn the building blocks of New Relic programmability: